diag_manager/fms_diag_file_object.F90

```diff
@@ -908,21 +953,21 @@ subroutine write_time_metadata(this)
 
   if (diag_file%time_ops) then
     call register_variable_attribute(fileobj, time_var_name, "bounds", &
-      trim(time_var_name)//"_bounds", str_len=len_trim(time_var_name//"_bounds"))
+      trim(time_var_name)//"_bnds", str_len=len_trim(time_var_name//"_bnds"))
```

Comment:
Is this to get it in like with the current?

Reply:
yes, it should be `_bnds`